docs/Examples.md

```diff
@@ -445,6 +452,33 @@ spec:
         - "/mnt/apps/app1.tgz"
         - "/mnt/apps/app2.tgz"
 ```
+In the above example, app1 and app2 are installed on Splunk Standalone instances.
+
+Example: Cluster Master
+
+```yaml
+apiVersion: enterprise.splunk.com/v1
+kind: ClusterMaster
+metadata:
+  name: cmexample
+  finalizers:
+  - enterprise.splunk.com/delete-pvc
+spec:
+  volumes:
+    - name: apps
+      configMap:
+        name: splunk-apps
+  defaults: |-
+    splunk:
+      apps_location:
+        - "/mnt/apps/app3.tgz"
+        - "/mnt/apps/app4.tgz"
+      apps_location_local:
+        - "/mnt/apps/app5.tgz"
+        - "/mnt/apps/app6.tgz"
+```
+
+In the above example, app3 and app4 are installed on Splunk Indexer instances of the Indexer Cluster for which it is Cluster Master. app5 and app6 are installed locally on the ClusterMaster instance.
```

Comment:
Move to 458:
In the ClusterMaster example, app3 and app4 are installed on any indexer instances that are managed by the cluster master. App5 and app6 are installed locally on the ClusterMaster instance.